### DISCUSSION TITLE
Location Area data for generation 7 and onwards

### DISCUSSION THREAD

**Discussion by totalaj** [PokeAPI/pokeapi#958] [Q&A] [answered]
Hi! I'm making an application using PokeAPI and so far it has been great, but I have noticed that locations in generation 7 and onwards don't have areas! Is this intentional, or am I missing something. If the data can be found somewhere else, I would love to know where. Cheers!

Edit: What I'm really looking for is pokemon encounters for those games. "Areas" specifically are not needed, but that's how I've gotten the encounters for the other gens.

**Comment by Naramsim** [comment, marked as answer]
Hello, most probably we lack the data. We would need to scrape it from somewhere and load it into our CSVs. 

Would you be able to help us?

**Reply by totalaj** [reply]
Absolutely, I'd love to! I could scrape Bulbapedia or something else for areas and encounters. I would need an example for how to format the CSV. Do y'all have a pipeline defined for how to do this?

**Reply by Naramsim** [reply]
There's no fixed pipeline. I think first you need to understand the data. https://github.com/PokeAPI/pokeapi/tree/master/data/v2/csv here there's many CSVs, you could take a look at the location_* ones. they are all interconnected with and ID system.
